scripts/countdown_erc721/README.md

```diff
@@ -120,3 +120,27 @@ This is a straightforward script that invokes the contract `purchase` function u
 ```sh
 ts-node ./scripts/countdown_erc721/4-purchase.ts 
 ```
+
+## Script 5: Set Metadata 
```

Comment:
@BravoNatalie can you add this latest script to the top of the file where you have the scripts listed?